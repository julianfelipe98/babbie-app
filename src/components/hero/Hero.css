#imageCellphoneInfo {
  padding-top: 25px;
  position: relative;
  width: 25rem;
  height: auto;
  filter: drop-shadow(0 0 5px rgba(0, 0, 0, 0.2));
  bottom: 10rem;
}
.hero-component {
  text-align: center;
}
