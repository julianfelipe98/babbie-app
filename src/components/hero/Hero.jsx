import React from "react";
import { Banner } from "../banner/Banner";
import phoneLogo from '../../assets/cp1.png';
import './Hero.css'
export const Hero = () => {
  const banner = {
    id: "1",
    title: "Aplicacion para la gestion veterinaria",
    subtitle: "Â¡El amor es una palabra de cuatro patas!",
    paragraph: "Descarga nuestra app en dispositivos android",
    styleBtn: "download-btn2",
    btn: "Descargala aqui",
  };
  return (
    <div className="hero-component">
      <Banner
        id={banner.id}
        title={banner.title}
        subtitle={banner.subtitle}
        paragraph={banner.paragraph}
        styleBtn={banner.styleBtn}
        btn={banner.btn}
      />
      <img id="imageCellphoneInfo" src={phoneLogo} alt="phone-logo" />
    </div>
  );
};
