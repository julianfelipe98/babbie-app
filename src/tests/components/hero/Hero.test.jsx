import { shallow } from "enzyme";
import { Hero } from "../../../components/hero/Hero";

describe("Hero component test", () => {
  test("should render <Hero/> in the correct way", () => {
    const wrapper = shallow(<Hero />);
    expect(wrapper).toMatchSnapshot();
  });
});
