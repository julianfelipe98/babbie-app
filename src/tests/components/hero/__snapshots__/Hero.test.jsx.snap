// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero component test should render <Hero/> in the correct way 1`] = `
<div
  className="hero-component"
>
  <Banner
    btn="Descargala aqui"
    id="1"
    paragraph="Descarga nuestra app en dispositivos android"
    styleBtn="download-btn2"
    subtitle="Â¡El amor es una palabra de cuatro patas!"
    title="Aplicacion para la gestion veterinaria"
  />
  <img
    alt="phone-logo"
    id="imageCellphoneInfo"
    src="cp1.png"
  />
</div>
`;
